language/src/rustspec_to_fstar.rs

```diff
@@ -890,6 +890,13 @@ fn translate_expression<'a>(
             ))
             .append(RcDoc::space())
             .append("end"),
+        Expression::FieldAccessor(e1, box (Field::TupleIndex(field), _)) => {
+            // TODO: this works only for tuples because for now, a
```

Comment:
Do you want to file a follow up for this todo?

Reply:
Oh, right, I forgot to do so!